lib/gpprmsya.gi

```diff
@@ -272,15 +272,39 @@ function ( G )
     return classes;
 end);
 
+BindGlobal("CheapIsomSymAlt",function(a,b)
+local hom;
+  if IsPermGroup(a) then
+    hom:=SmallerDegreePermutationRepresentation(a);
+    if Image(hom)=b then return hom;
+    elif NrMovedPoints(Range(hom))<NrMovedPoints(a) then
+      return hom*IsomorphismGroups(Image(hom),b);
+    else
+      return IsomorphismGroups(a,b);
+    fi;
+  fi;
```

Comment:
So if the first argument is not a perm group, this function suddenly returns nothing. That seems wrong. Perhaps it should return `fail`, or call `Error`?

Reply:
Changed,.